setup.py

```diff
@@ -0,0 +1,10 @@
+from setuptools import setup
+
+setup(name='flooddrake',
+      description='Firedrake DG shallow water model with wetting and drying for flood simulation',
+      url='https://github.com/firedrakeproject/flooddrake',
+      author='Alastair C. A. Gregory, Colin. J. Cotter',
+      author_email='alsgregory@aol.com',
+      license='Imperial College London',
+      packages=['flooddrake'],
+      zip_safe=False)
```

Comment:
Again, I apologise for these setup errors - I've never done one before and haven't a clue what these options mean. But no, this package would be easy and 'safe' to zip.